python/README

```diff
@@ -0,0 +1,18 @@
+20170511 
+
+Building with the automake-generated Makefile succeeds, but 
```

Comment:
Better to file issues for build problems rather than using notes in a README file like this.